recipes/scikit-fmm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "scikit-fmm" %}
+{% set version = "2019.1.30" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb64b6d8e30b8df8f8636d5fc4fd7ca6a9b05938ccd62518c80c1d9e823069dd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
I assume this uses the Numpy C API if so: http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

If not you probably don't need `numpy` here.

Reply:
I tired this in https://github.com/conda-forge/staged-recipes/pull/8362/commits/6e2d2276e90676e6ecebbef656ebf40c33ae7a9c - but then the setup fails.

Reply:
Then you're probably using the C API and should use `pin_compatible` in `run` (see above link for details).

Reply:
As far as I see it, it is only importing `from numpy.distutils.core import setup` I do not think it is necessary to pin then, or?